### ISSUE TITLE
local network card doesn't load

### ISSUE THREAD

**Issue by legend069** [seud0nym/tch-gui-unhide#137]
updated one of them to tch-gui-unhide Version 2023.02.02@16:30
it's on one of the bridged devices.
the ```local network tab``` doesn't seem to load when you click on the ```settings/house icon``` 
comes up with the green spinning wheel and that's about it.

**Comment by seud0nym** [comment]
Hi.

Yes, it looks like I have introduced a bug when it is in bridged mode.

Try these commands:
```
sed -e 's/tonumber(content\["dhcplogv6"\])/(& or 0)/' -i /www/docroot/modals/ethernet-modal.lp
/etc/init.d/nginx restart
```

**Comment by legend069** [comment]
> 

running the commands seems to of fixed it.

**Comment by willat8** [comment]
I applied the same on my bridged device (running 18.1) but it hasn't helped.

**Comment by seud0nym** [comment]
Could you please post the output from:
```
logread -e nginx -l 100
```
after it happens? Thanks.

**Comment by willat8** [comment]
Sure:

```
Tue Feb  7 20:05:17 2023 daemon.err nginx: 2023/02/07 20:05:16 [error] 1991#0: *58 lua entry thread aborted: runtime error: [string "/modals/ethernet-modal.lp"]:308: attempt to compare nil with number stack traceback: coroutine 0: 	[string "/modals/ethernet-modal.lp"]:308: in function 'content' 	/usr/lib/lua/web/web.lua:249: in function 'process' 	content_by_lua(nginx.conf:114):4: in function <content_by_lua(nginx.conf:114):1>, client: 192.168.0.101, server: localhost, request: "GET /modals/ethernet-modal.lp HTTP/1.1", host: "192.168.0.9", referrer: "http://192.168.0.9/gateway.lp"
```

**Comment by seud0nym** [comment]
Hmm. So, very strange. Line 308 doesn't do a compare.

The fix above is a bit of a hack just to get it going. I have done a proper fix and put out a [pre-release](https://github.com/seud0nym/tch-gui-unhide/releases/tag/2023.02.08) so you can try that:
```
./tch-gui-unhide -u && ./tch-gui-unhide -y
```
Let me know how you go.

**Comment by willat8** [comment]
Looking good now. Thanks!

**Comment by seud0nym** [comment]
Fixed in 2023.02.23
